conftest.py

```diff
@@ -30,6 +31,13 @@ def set_seed(seed: int = 0):
     yield
 
 
+@contextmanager
```

Comment:
Why contextmanager ?

Reply:
to make it work like an annotation (like we did for set_seed). Have you other option in mind?